src/Illuminate/Foundation/Testing/TestCase.php

```diff
@@ -114,6 +113,26 @@ protected function refreshApplication()
         $this->app = $this->createApplication();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function runTest(): mixed
+    {
+        $result = null;
+
+        try {
+            $result = parent::runTest();
+        } catch (Throwable $e) {
+            if (! is_null(static::$latestResponse)) {
+                static::$latestResponse->transformNotSuccessfulException($e);
+            }
+
+            throw $e;
+        }
+
+        return $result;
+    }
+
```

Comment:
For all who are looking for PHPStorm `Jump to source` fixes because of this update and found my comment - just upgrade your deps to PHPUnit 10